src/httpd_dacp.c

```diff
@@ -1235,10 +1235,10 @@ dacp_reply_cue_play(struct httpd_request *hreq)
       else
 	{
 	  /* Play from Up Next queue */
-	  if (status.status == PLAY_STOPPED && pos > 0)
```

Comment:
when `PLAY_STOPPED` the `status.item_id` == `0` which means that we can simply obtain the queue item based on the `pos` index.